capstone/src/main/webapp/form.html

```diff
@@ -11,8 +11,19 @@
   <body>
     <div id="content">
       <p><h1>Quiz</h1></p>
-      <iframe src="https://docs.google.com/forms/d/e/1FAIpQLScrpUcq5Jnv6knV6K3aexWa1m972EmMzHJUvXulHCfBsJaEpw/viewform?embedded=true">Loading…</iframe>
-      <a href="https://docs.google.com/forms/d/1JAOSyrHmloNFiUibqhMLRQj1qcDDhHr0-nDhZ2ouQJM/edit"> For Teacher </a> 
+      <button id="authorize_button" style="display: none;">Authorize</button>
+      <button id="signout_button" style="display: none;">Sign Out</button>
+      <button onclick="callScriptFunction()">Create Form</button>
     </div>
+
+    <pre id="content" style="white-space: pre-wrap;"></pre>
+    <script src = "form-loader.js"></script>
+    <script async defer src="https://apis.google.com/js/api.js"
+      onload="this.onload=function(){};handleClientLoad()"
+      onreadystatechange="if (this.readyState === 'complete') this.onload()">
+    </script>
```

Comment:
This is also fine, but since we are not crazy concerned with performance until it becomes a problem, this can be simplified to a script in head. (Has to be after form-loader.js or else the handleClientLoad function might not be defined yet.

<script src="https://apis.google.com/js/api.js" onload="handleClientLoad()"</script>

Reply:
Sounds good